src/language/predicates.js

```diff
@@ -35,15 +35,15 @@ export function isSelectionNode(node: ASTNode): boolean %checks {
 
 export function isValueNode(node: ASTNode): boolean %checks {
   return (
+    node.kind === Kind.VARIABLE ||
     node.kind === Kind.INT ||
     node.kind === Kind.FLOAT ||
     node.kind === Kind.STRING ||
     node.kind === Kind.BOOLEAN ||
     node.kind === Kind.NULL ||
     node.kind === Kind.ENUM ||
     node.kind === Kind.LIST ||
-    node.kind === Kind.OBJECT ||
-    node.kind === Kind.OBJECT_FIELD
```

Comment:
`OBJECT_FIELD` is lumped with Values in `kinds.js`, but that's because it doesn't really belong anywhere else. However, if I give an `ObjectFieldNode` to a function expecting a `ValueNode`, we've messed something up.